### ISSUE TITLE
install script fails on macos - should check dependencies (pipx)

### ISSUE THREAD

**Issue by planetf1** [containers/ramalama#486]
Tried the script based install on macOS (sonama)
```
curl -fsSL https://raw.githubusercontent.com/containers/ramalama/s/install.sh | bash
``

which failed with:
```
curl -fsSL https://raw.githubusercontent.com/containers/ramalama/s/install.sh | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  2937  100  2937    0     0  14249      0 --:--:-- --:--:-- --:--:-- 14188
main: line 50: pipx: command not found
```

I installed pipx (in my case via `brew install pipx`) which was then fine.

One challenge with python tools is always the dependencies & clashes on a system. For example I try not to install global packages, and prefer to keep pretty much everything within a virtual environment

**Comment by ericcurtin** [comment]
Yeah this is something we need to work on. Need to try and find a way of create an easy and seamless install experience across macOS and various Linux distros. I also would like if the user didnt explicitly have to set up a venv, but if the solution implicitly set up a venv under the hood, I think it would be ok.

Can we just add brew install pipx? Im travelling dont have macbook with me
